tensorflow/compiler/xla/pjrt/gpu_device.cc

```diff
@@ -61,10 +61,11 @@ xla::StatusOr<xla::DeviceAssignment> GpuClient::GetDefaultDeviceAssignment(
 // Builds an xla::LocalClient for the GPU platform.
 StatusOr<LocalClient*> GetGpuXlaClient() {
   TF_ASSIGN_OR_RETURN(se::Platform * platform,
-                      PlatformUtil::GetPlatform("CUDA"));
+                      PlatformUtil::GetPlatform("gpu"));
```

Comment:
This is essentially the only change in this PR.

in `tensorflow/compiler/xla/service/platform_util.cc` `CanonicalPlatformName`
`gpu` will ultimately be replaced by eiter `cuda` (like before) or `rocm`, if we are compiling with `TENSORFLOW_USE_ROCM`

Reply:
Seems reasonable to me. (I sort of wish the design just gave them different names and allowed them to be linked simultaneously, but that's not today's problem.)